src/main/java/de/tum/in/www1/artemis/domain/Result.java

```diff
@@ -59,10 +59,6 @@
     @JsonView(QuizView.After.class)
     private Boolean successful;
 
-    @Column(name = "build_artifact")
```

Comment:
do not delete, but deprecate it for easier migration

Reply:
I reverted the change